Sources/BraintreeDropIn/Models/BTUIKCardType.m

```diff
@@ -163,7 +163,11 @@ + (NSArray *)allCards
                                                                 prefixes:@[@"^(5[1-5]|222[1-9]|22[3-9]|2[3-6]|27[0-1]|2720)\\d*"]];
         BTUIKCardType *discover = [[BTUIKCardType alloc] initWithBrand:BTDropInLocalizedString(CARD_TYPE_DISCOVER)
                                                       securityCodeName:BTDropInLocalizedString(CID_FIELD_PLACEHOLDER)
-                                                              prefixes:@[@"^(6011|65|64[4-9]|622)\\d*"]];
+                                                              prefixes:@[@"^(6011|65|64[4-9]|622)\\d*"]
+                                                       relaxedPrefixes:nil
+                                                    validNumberLengths:[NSIndexSet indexSetWithIndexesInRange:NSMakeRange(16, 4)]
```

Comment:
How come this is 16 instead of 19?

Reply:
I believe this makes a range starting from 16 with length 4 (so 16-19)

Reply:
Oh, I see! That makes sense.